wp-api.js

```diff
@@ -237,35 +222,27 @@
 	wp.api.models.User = BaseModel.extend(
 		/** @lends User.prototype  */
 		{
-			idAttribute: 'ID',
+			idAttribute: 'id',
 
 			urlRoot: WP_API_Settings.root + '/users',
 
 			defaults: {
-				ID: null,
-				username: '',
+				id: null,
+				avatar_url: '',
+				capabilities: {},
+				description: '',
 				email: '',
-				password: '',
-				name: '',
+				extra_capabilities: {},
 				first_name: '',
 				last_name: '',
+				link: '',
+				name: '',
 				nickname: '',
+				registered_date: new Date(),
+				roles: [],
 				slug: '',
-				URL: '',
-				avatar: '',
-				meta: {
-					links: {}
-				}
-			},
-
-			/**
-			 * Return avatar URL
-			 *
-			 * @param {number} size
-			 * @returns {string}
-			 */
-			avatar: function( size ) {
-				return this.get( 'avatar' ) + '&s=' + size;
+				url: '',
+				username: ''
```

Comment:
@tlovett1 Any reason for not including `_links` here?  Saw you had it included for Terms, Posts, etc.